egs/swbd/s5c/local/nnet3/run_lstm.sh

```diff
@@ -19,36 +31,43 @@ hidden_dim=1024
 recurrent_projection_dim=256
 non_recurrent_projection_dim=256
 chunk_width=20
-chunk_left_context=20
-clipping_threshold=5.0
+chunk_left_context=40
```

Comment:
OK.  It might be worthwhile looking again at increasing the frames-per-eg, then (e.g. to 30?), and maybe reducing the minibatch size or learning rate slightly if you worry about stability (at least, until @xiaohui-zhang  checks in his changes to the max-change code).  It just seems to me that having the left-context much more than the frames-per-eg is a bit wasteful.